neptune/new/metadata_containers/project.py

```diff
@@ -465,3 +466,23 @@ def sync(self, wait: bool = True) -> None:
             https://docs.neptune.ai/api-reference/project#.sync
         """
         return MetadataContainer.sync(self, wait=wait)
+
+    def trash_objects(
+        self,
+        ids: Union[str, Iterable[str]],
+    ) -> None:
+        """TODO: add docs"""
+        if self._mode == Mode.READ_ONLY:
+            logger.warning("Client in read-only mode, nothing will be saved to server.")
```

Comment:
That's something unusual, right? I don't think that we would like to have it and it should be handled by the implementation of `NeptuneBackend`